src/models/guild_preference.ts

```diff
@@ -67,17 +67,20 @@ export default class GuildPreference {
         }
         if (gameOptionModified) {
             this.updateGuildPreferences(dbContext.kmq);
+            this.updateGameSession(false);
```

Comment:
Game session doesn't exist at this point, not needed here

Reply:
Thanks!